lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -174,11 +173,7 @@ WasmBinaryReader::ReadSectionHeader()
 
 #if ENABLE_DEBUG_CONFIG_OPTIONS
     Assert(idSize < 64);
-    char16 buf[64];
-    size_t convertedChars = 0;
-    mbstowcs_s(&convertedChars, buf, idSize + 1, sectionName, _TRUNCATE);
-    buf[idSize] = 0;
-    TRACE_WASM_SECTION(_u("Section Header: %s, length = %u (0x%x)"), buf, sectionSize, sectionSize);
+    TRACE_WASM_SECTION(_u("Section Header: %s, length = %u (0x%x)"), sectionName, sectionSize, sectionSize);
```

Comment:
`sectionName` is a `char*` does it work in with a printf function using a char16 format string ?